.github/templates/pr.yml

```diff
@@ -39,12 +39,11 @@ jobs:
   _: #@ template.replace(runTests("UWP Managed", runSyncTests = False, additionalSecrets = ["Pfx_Password", "Base64_Encoded_Pfx"]))
   _: #@ template.replace(runNetCoreTests('["net6.0"]'))
   _: #@ template.replace(runTests("macOS", runSyncTests = False))
-  _: #@ template.replace(runTests("iOS", runSyncTests = False))
+  _: #@ template.replace(runTests("iOS", runSyncTests = True))
   _: #@ template.replace(runTests("Android", runSyncTests = False, additionalSecrets=["AWS_DEVICEFARM_ACCESS_KEY_ID", "AWS_DEVICEFARM_SECRET_ACCESS_KEY", "DEVICEFARM_PROJECT_ARN", "DEVICEFARM_ANDROID_POOL_ARN"]))
   _: #@ template.replace(runWovenClassesTests())
   _: #@ template.replace(runSourceGenerationTests())
   test-weaver:
     uses: ./.github/workflows/test-weaver.yml
     name: Test
-  _: #@ template.replace(runTests("Code Coverage"))
-  _: #@ template.replace(cleanupBaas(["Code Coverage"]))
```

Comment:
Need to restore this before merging

Reply:
Which one? It seems that you've done this. If so you can resolve the comment.

Reply:
Oh yeah, strange that it didn't mark the comment as outdated.